src/test/java/team/serenity/logic/parser/DelLsnCommandParserTest.java

```diff
@@ -1,11 +1,31 @@
 package team.serenity.logic.parser;
 
+import static team.serenity.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static team.serenity.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static team.serenity.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
 import org.junit.jupiter.api.Test;
 
+import team.serenity.logic.commands.lesson.DelLsnCommand;
+import team.serenity.logic.parser.lesson.DelLsnCommandParser;
+import team.serenity.model.group.GroupName;
+import team.serenity.model.group.lesson.LessonName;
+
 class DelLsnCommandParserTest {
 
     @Test
-    void parse() {
+    void parse_emptyInput_throwParseException() {
+        DelLsnCommandParser parser = new DelLsnCommandParser();
+        assertParseFailure(parser, "     ",
```

Comment:
Instead of hardcoding this, perhaps you can reference `PREAMBLE_WHITESPACE` in `CommandTestUtil` class?

Similar comment for the other test cases in this file :)

Reply:
Resolved for `Dellsn` and `ViewGrp`